tests/std/tests/P0811R3_midpoint_lerp/test.cpp

```diff
@@ -1023,6 +1023,15 @@ bool test_lerp() {
     return true;
 }
 
+void test_gh_2112() {
+    // GH-2112 <cmath>: std::lerp is missing Arithmetic overloads
+    assert(lerp(int{}, int{}, int{}) == 0.0);
+    STATIC_ASSERT(lerp(int{}, int{}, int{}) == 0.0);
+
+    assert(lerp(float{}, float{}, double{}) == 0.0);
+    STATIC_ASSERT(lerp(float{}, float{}, double{}) == 0.0);
+}

```

Comment:
Suggestion: Although this is small (only 2 test cases), it would be more consistent with the other test functions to make this a `constexpr bool` function that `main()` tests at both compiletime and runtime, allowing the `STATIC_ASSERT` repetition here to be dropped.